.github/workflows/ubuntu_20.04/build.sh

```diff
@@ -51,7 +51,9 @@ CXXFLAGS="-std=c++17 -O1 $ARCH_FLAGS" CFLAGS="-O1 $ARCH_FLAGS" ./configure --pre
     --with-kea=/usr/bin/kea-config \
     --with-tiledb \
     --with-crypto \
-    --with-ecw=/opt/libecwj2-3.3
+    --with-ecw=/opt/libecwj2-3.3 \
+    --with-zarr \
```

Comment:
would be nice if --with-zarr could take a non default install location

Reply:
What do you mean, we can pass any directory to `--with-zarr` already? Or do you mean in the CI specifically?

Reply:
Sorry, I missed that part. I did mean that the non-standard path can be selected by the user outside of the CI.
However for checking for the lib/headers in default location you should use something similar to https://github.com/OSGeo/gdal/blob/64db832b3e475bb70d5acae99820c5ee405c9838/gdal/configure.ac#L2397 instead of looking up a hard-coded path. Namely -I/usr/include should never be added manually to the CFLAGS